src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
```

Comment:
why would you set this value? it should be defined by paddings and content

Reply:
I set it because I thought it should have been set, since in the task it is said that 
_"cards should have fixed **width** and fixed distances between them"_